backend/app/core/review_helpers.py

```
@@ -0,0 +1,92 @@
+동작 변화 없음 — 함수 시그니처/로직은 라우터 원본과 동일.
+"""
+
+from __future__ import annotations
+
+import random
+
+from sqlalchemy.ext.asyncio import AsyncSession
+
+
+async def get_seller_product_ids(
+    db: AsyncSession,
```

Ruff (0.15.12)

[warning] Unused function argument: `db`

(ARG001)